src/BoundingBoxCameraSensor.cc

```diff
@@ -383,6 +383,12 @@ bool BoundingBoxCameraSensor::Update(
     return false;
   }
 
+  if(this->HasInfoConnections())
```

Comment:
gazebo coding style nit: add space between `if` and `(`. There are also a few other places in this PR.